bootstrap/lib/cache/index.js

```diff
@@ -108,7 +117,8 @@ const withCache = (execute, options) => {
             // Add some entropy here because of possible scenario where the key won't be set before multiple
             // other instances in a burst request try to access the coalescing key.
             const randomMs = Math.random() * options.requestCoalescing.entropyMax
-            await delay(randomMs)
+            // Adding this next condition because of mocha timeout issues
+            if (randomMs) await delay(randomMs)
```

Comment:
I would probably change the condition to skip if `!options.requestCoalescing.entropyMax`, it reads a little better than checking if the value of `randomMs` exists or not

Reply:
I agree. Did just quickly just to make the test pass. I would like to get rid of this condition all together.